### ISSUE TITLE
The loss does not converge

### ISSUE THREAD

**Issue by BigBinnie** [tegg89/SRCNN-Tensorflow#55]
I got the result that the loss does not converge.If someone has the same result or knows how to solve it.

**Comment by BigBinnie** [comment]
Now i have solved it

**Comment by Whitecatwarrior** [comment]
hello,i have got the same problem
`
Epoch: [11882], step: [926720], time: [92697.6400], loss: [0.00161069]

Epoch: [11882], step: [926730], time: [92698.6210], loss: [0.00290118]

Epoch: [11882], step: [926740], time: [92699.6220], loss: [0.00086975]

Epoch: [11882], step: [926750], time: [92700.6011], loss: [0.00169979]

Epoch: [11882], step: [926760], time: [92701.6095], loss: [0.00303134]

Epoch: [11882], step: [926770], time: [92702.6365], loss: [0.00131717]

Epoch: [11882], step: [926780], time: [92703.6965], loss: [0.00106838]
`
could u please share me with your solution

**Comment by BigBinnie** [comment]
Sorry, I couldn't remember the details of this problem, but the codes should be right. It seems that you have trained many epochs, so it might be reasonable that the loss doesn't decrease anymore. I think you could try to evaluate the current checkpoint to check if you have got a result close to the paper.

If the problem couldn't be solved by the above method, I have a repo forked from this. You could try codes that I'm sure about rightness in https://github.com/BigBinnie/SRCNN.

**Comment by Whitecatwarrior** [comment]
Thank you very much for your reply，I'll try what you say
